src/configfile.ts

```diff
@@ -215,6 +218,26 @@ interface ChromeConfig extends BrowserConfigBase {
    * @TJS-minimum 1
    */
   cpuThrottlingRate?: number;
+
+  /**
+   * Optional config to turn on performance tracing.
+   */
+  trace?: TraceConfig|boolean;
```

Comment:
Maybe this could be `TraceConfig|true`, and then you wouldn't need to handle the `false` case below?

Reply:
Oh nice! Yup, that works